### ISSUE TITLE
Crash when playing HLS on Android Emulator level 29

### ISSUE THREAD

**Issue by albilaga** [androidx/media#909]
### Version

Media3 1.2.0

### Devices that reproduce the issue

- Emulator running API 29 on mac with M series processor

### Devices that do not reproduce the issue

- Xiaomi Redmi 9c running Android 10 (API 29 also)

### Reproducible in the demo app?

Yes

### Reproduction steps

1. Run demo app
2. Go to HLS
3. Choose the first one
4. App will stuck in there and not showing anything and emulator will be frozen
5. Emulator will crash after long period of frozen

### Expected result

The media should plays successfully

### Actual result

Player frozen and emulator also frozen. Here is the log
[error.log](https://github.com/androidx/media/files/13717180/error.log)

### Media

Media is the same as in the demo and HLS

### Bug Report

- [X] You will email the zip file produced by `adb bugreport` to android-media-github@google.com after filing this issue.

**Comment by tianyif** [comment]
Hi @albilaga,

Happy new year and thanks for reporting! From your log, I saw the playback was trying to start and the last piece of log from the demo app is 

```
{
      "header": {
        "logLevel": "DEBUG",
        "pid": 3895,
        "tid": 3895,
        "applicationId": "androidx.media3.demo.main",
        "processName": "androidx.media3.demo.main",
        "tag": "EventLogger",
        "timestamp": {
          "seconds": 1702998191,
          "nanos": 433000000
        }
      },
      "message": "audioTrackInit [eventTime\u003d5.64, mediaPos\u003d0.00, window\u003d0, period\u003d0, 2,12,22050,false,false,22052]"
    },
```
I haven't got a solid idea on what was going on after it though.

Did you see this issue on a emulator with the specific definition? I'm using M1 mac and couldn't reproduce the issue with Pixel 5, Pixel 6 and Nexus 4. And all of them are with API 29.

**Comment by albilaga** [comment]
@tianyif do you test on devices or in emulator? emulator always fail for me but devices all ok. emulator always like this
![CleanShot 2024-01-04 at 16 19 01@2x](https://github.com/androidx/media/assets/2399099/a65f654a-348e-4dfd-b70a-d525ef648201)

**Comment by tianyif** [comment]
Yep, I tested on the emulators of Pixel 5, Pixel 6 and Nexus 4.

**Comment by thisisthekap** [comment]
@tianyif Is it working properly for you in the emulator?

**Comment by albilaga** [comment]
hmm.. there is something wrong with my emulator then. But not really sure which one is wrong also

**Comment by tianyif** [comment]
@thisisthekap yes, they were working properly on the emulators when I ran. Did you see the same issue?

**Comment by thisisthekap** [comment]
@tianyif Yes, same as @albilaga

**Comment by thisisthekap** [comment]
@tianyif Have you gained any further insight on that one?
